src/index.ts

```diff
@@ -219,9 +223,18 @@ export const build = <FactoryResultType>(
 
     return buildTimeMapFunc(postBuild(afterTraitPostBuildFields));
   };
+
   builder.reset = () => {
     sequenceCounter = 0;
   };
+
+  builder.many = (
+    times: number,
+    buildTimeConfig: BuildTimeConfig<FactoryResultType>
+  ): FactoryResultType[] => {
+    return new Array(times).fill(0).map((_) => builder(buildTimeConfig));
```

Comment:
Like this, you can avoid the `fill/map` trick:

```suggestion
    return Array.from({ length: times }, () => builder(buildTimeConfig));
```